docs/framework/network-programming/tls.md

```diff
@@ -83,7 +79,7 @@ The remainder of this topic is not relevant when targeting .NET Framework 4.7 or
 
 <a name="wcf-tcp-cert"></a>
 
-### For WCF TCP transport using transport security with certificate credentials
+### For WCF using TCP transport security with Certificate Credentials
```

Comment:
is Certificate Credentials the name of a feature or something? Otherwise, we should be using sentence case on headings. I probably missed the other heading when changing this on the draft,

Reply:
lower case is better, it's not a feature of the product, but the protocol.

Reply:
BTW: I picked the casing used most. Happy to change it all to lower case if it is desired.
@mconnew @Lxiamail @zhenlan can you please comment?

Reply:
Probably my bad on not using lowercase throughout the doc.

Reply:
Fixed all instances.